packages/nimble-components/build/generate-workers/index.ts

```diff
@@ -33,7 +33,7 @@ const sourceCode: string = fs.readFileSync(modulePath, 'utf-8');
 const workerTypesPath: string = resolveModulePath(workerTypes);
 const typesCode: string = fs.readFileSync(workerTypesPath, 'utf-8');
 
-const fileContent: string = `// eslint-disable-next-line no-template-curly-in-string
+const fileContent = `// eslint-disable-next-line no-template-curly-in-string
```

Comment:
This eslint-disable looks strange to me. It's not applying to the contents of this file, it's being emitted into the generated file. Do we actually lint the generated files? If so, could we put the disable inside the source contents instead? Or could we just lint the source files? Or even better, could we fix the code not to need to violate this rule?

All of this is pre-existing so it doesn't need to be fixed in this PR, just pointing it out in case there's an easy fix or you think it's worth tracking as tech debt / submitting a follow up PR.

Reply:
it's fixed in this pending PR https://github.com/ni/nimble/pull/2154